### PR TITLE
generate: optimize namespace setup log and fix manpage

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: man/ocitools-generate.1.md [hunk 1]
```diff
@@ -45,10 +45,11 @@ read the configuration from `config.json`.
...
 
-**--cgroup**=[*PATH*]
-  Use a Cgroup namespace.  If *PATH* is set, join that namespace.  If it
-  is unset, create a new namespace.  The special *PATH* `host` removes
-  any existing Cgroup namespace from the configuration.
+**--cgroup**=*PATH*
+  Use a Cgroup namespace. *PATH* to file of an existing Cgroup namespace.
+  If *PATH* is set, join that namespace. The sepcial *PATH*  ""  set to
+  create a new namespace.  The special *PATH* `host` removes any existing
+  Cgroup namespace from the configuration.
```
Comment (wking) [inline man/ocitools-generate.1.md]: There's a “sepcial” → “special” typo in here.  And the first two sentences read a bit awkardly to me.  How about:

> Use a Cgroup namespace where *PATH* is an existing Cgroup namespace file to join.  The special empty-string *PATH*  creates a new namespace.  The special *PATH* `host` removes any existing Cgroup namespace from the configuration.

File: cmd/ocitools/generate.go [hunk 2]
```diff
@@ -366,19 +360,12 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
...
 
-func checkNs(nsMaps map[string]string, nsName string) bool {
-	if _, ok := nsMaps[nsName]; !ok {
-		return false
-	}
-	return true
-}
-
-func setupLinuxNamespaces(g *generate.Generator, needsNewUser bool, nsMaps map[string]string) {
+func setupLinuxNamespaces(g *generate.Generator, needsNewUser bool, context *cli.Context) {
```
Comment (wking) [inline cmd/ocitools/generate.go]: I'd rather have [`context` first][1], but that isn't a big deal ;).

[1]: https://github.com/opencontainers/ocitools/pull/159#discussion_r72564169

File: man/ocitools-generate.1.md (hunk 1 above)
Comment (Mashimiao) [inline man/ocitools-generate.1.md]: On 08/02/2016 12:39 AM, W. Trevor King wrote:
> There's a “sepcial” → “special” typo in here. And the first two sentences read a bit awkardly to me. How about:
> 
>     Use a Cgroup namespace where /PATH/ is an existing Cgroup namespace file to join. The special empty-string /PATH/ creates a new namespace. The special /PATH/ |host| removes any existing Cgroup namespace from the configuration.
> 

Thx, fixed.

**Comment by wking** [comment]
15f529b looks good to me.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**